### ISSUE TITLE
Can the DOA algorithm here be applied to ReSpeaker USB Mic Array (4 Mic with 1 ASR & Playback)?

### ISSUE THREAD

**Issue by Crysple** [respeaker/mic_array#44]
My device is [ReSpeaker USB Mic Array](https://wiki.seeedstudio.com/ReSpeaker-USB-Mic-Array/#extract-voice).
In that page, it said that I could extract VOA from the hardware. But for some kind of reason, we have to calculate the DOA by channel data. So can the [code](https://github.com/respeaker/mic_array/blob/2fbbe51f4243de0329fa5d8d63c9fca7638f45ff/mic_array.py#L88) here be used to calculate DOA? I also saw other versions [here](https://github.com/voice-engine/voice-engine/tree/master/voice_engine), but the relationship between them and [hardwares](https://wiki.seeedstudio.com/ReSpeaker-USB-Mic-Array) is not that clear. The most likely one is [6p1](https://github.com/voice-engine/voice-engine/blob/master/voice_engine/doa_respeaker_6p1_mic_array.py), but actually the hardware has only 4 raw mics. The other two are ASR & Playback.
In a nutshell, I am confusing that can I use the algo here, if so, which one should I use?

Thank you
